uvicorn/protocols/http/httptools_impl.py

```
@@ -478,7 +484,7 @@ async def send(self, message: ASGISendEvent) -> None:
             status_code = message["status"]
             headers = self.default_headers + list(message.get("headers", []))
 
-            if CLOSE_HEADER in self.scope["headers"] and CLOSE_HEADER not in headers:
+            if has_connection_close(self.scope["headers"]) and not has_connection_close(headers):
                 headers = headers + [CLOSE_HEADER]
```

Ruff (0.16.1)

[warning] Consider `[*headers, CLOSE_HEADER]` instead of concatenation

Replace with `[*headers, CLOSE_HEADER]`

(RUF005)